client/signup/config/flows-pure.js

```diff
@@ -271,6 +271,17 @@ export function generateFlows( { getSiteDestination = noop, getPostsDestination
 		};
 	}
 
+	if ( config.isEnabled( 'signup/import-landing-handler' ) ) {
+		flows[ 'from-site' ] = {
+			steps: [ 'import-from-url', 'user', 'domains' ],
+			destination: () => '/checklist',
+			description: 'A flow to kick off an import during signup',
+			disallowResume: true,
+			allowContinue: false,
```

Comment:
This doesn't do anything - I think it's a c/p typo introduced in #20024 from the `autoContinue` param introduced in #9682.

Reply:
Def copy pasta -- removed